include/deal.II/base/mg_level_object.h

```diff
@@ -137,6 +137,19 @@ class MGLevelObject : public Subscriptor
   unsigned int max_level () const;
 
   /**
+   * Apply the action @p action to every object stored in here. The
+   * parameter @p action is expected to be a function object that accepts
+   * the syntax
+   * <code>
+   *   action(const unsigned int level, Object &object);
+   * </code>
+   * This means this function can accept a lambda, a std::function, or a plain
+   * function pointer.
```

Comment:
Mainly for performance reasons (not that it matters much). Thanks for letting me merge as is.